specs/app_settings/01_main.rst

```diff
@@ -0,0 +1,236 @@
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Problem Definition
+---------------------
+
+- There are some app features that we need to provide the user access to.
+    - Access Terms of Use
+    - Access Proviacy Policy
+    - Change password
+    - Enable/disable biometric authentication
+    - Option to Give feedback (Send us an email)
+    - Option to rate the app in the store.
+    - View app version (For bug reporting)
+    - View used open source licenses
+
+
+Inputs
+-----------
+
+ - Depending on the use case: Password and biometrics information, Terms, Privacy Policy
+
+
+Outputs
+----------
+
+Outputs depend on the selected option.
+- Open webpage (Terms & Privacy Policy)
+- Start change password flow
+- Biometric auth has been enabled / disabled
+- Open up email program (Give feedback)
+- Open store for rating / writing review (Rate app)
+- App version is displayed
+- View used open source licenses
+
+
+Use Case Scenarios
+-----------------------
+
+App version
+~~~~~~~~~~~~~~~~
+
+Preconditions
++++++++++++++
+
+- App is set up with a password
+
+Steps
++++++
+
+1. Open app settings
+
+Postconditions
+++++++++++++++
+
+- User can see app version in way so we know which version they are using.
+
+
+View Terms of Use
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Inherits from `App version`_
+
+Steps
++++++
+
+2. Open Terms of Use
+3. Browser opens with https://safe.gnosis.io/terms
+4. User goes back
+
+Postconditions
+++++++++++++++
+
+- User is back on the settings page.
+
+
+View Privacy Policy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Inherits from `View Terms of Use`_
+
+Steps
++++++
+
+2. Open Privacy Policy
+3. Browser opens with https://safe.gnosis.io/privacy
+4. User goes back
+
+
+Start change password flow
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Inherits from `App version`_
+
+Steps
++++++
+
+2. Open "Change password"
+
+Postconditions
+++++++++++++++
+
+- "Change password" flow is started
+
+
+Enable biometric auth
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- *iOS: "Touch ID" or "Face ID"*
+- *Android: "Fingerprint" or "Face ID"*
+
+Preconditions
++++++++++++++
+
+- App is set up with a password
+- Biometric auth is disabled
+
+Steps
++++++
+
+1. Open app settings
+2. Enable biometric auth
+3. User has to confirm with biometric auth
+
+Postconditions
+++++++++++++++
+
+- Biometric auth is enabled.
+- User is still on settings page.
+
+
+Disable biometric auth
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+- *iOS: "Touch ID" or "Face ID"*
+- *Android: "Fingerprint" or "Face ID"*
+
+Preconditions
++++++++++++++
+
+- App is set up with a password
+- Biometric auth is enabled
+
+Steps
++++++
+
+1. Open app settings
+2. Disable biometric auth
+
+Postconditions
+++++++++++++++
+
+- Biometric auth is disabled.
+- User is still on settings page.
+
+
+Rate app
+~~~~~~~~~~~~~
+
+Inherits from `App version`_
+
+Steps
++++++
+
+2. Open "Rate app"
+
+Postconditions
+++++++++++++++
+
+- Store is opened on a screen where user can rate and review the Gnosis Safe.
```

Comment:
Should we have the same go back logic as for the terms?

Reply:
Yea, not sure if we need to specify that. I think on iOS the app store is opened, so going back is harder anyway.